app/resources/css-default.template

```diff
@@ -177,11 +177,16 @@ div.languages {
   text-decoration: none;
   color: black;
   text-align: {{end}};
+  float:right;
+  margin-right:5%;
```

Comment:
Without margin it looks like this
![screencapture-localhost-8000-global-home-html-1491375611176](https://cloud.githubusercontent.com/assets/7677962/24693547/e7a8b90e-19ff-11e7-87d9-12dd67bccd9d.png)

without margin and with padding-right:30px,it look like this
![screencapture-localhost-8000-global-home-html-1491375636756](https://cloud.githubusercontent.com/assets/7677962/24693566/000c8e80-1a00-11e7-9d80-2c591e01bf67.png)